docs/helm-chart-checks.md

```diff
@@ -23,7 +24,7 @@ Helm chart checks are categorized into the following types:
 > **_NOTE:_**  The current release of Helm charts verify only the mandatory type of checks.
```

Comment:
**Types of Helm chart checks / Table 1**
**Recommended Description**
Consider rephrasing to "Checks are about to become mandatory; we recommend fixing any check failures."

Reply:
Done.